templates/cluster-template-persistent-example.yaml

```diff
@@ -0,0 +1,223 @@
+apiVersion: cluster.x-k8s.io/v1beta1
```

Comment:
Name this file cluster-template-cdi.yaml? And we may need an internal version of this file?

Reply:
what is internal version for?

Reply:
Currently we have two templates, standard (external infra) and internal infra version. For this CDI version template, we should have both variants too?